tests/hf/test_tapering.py

```diff
@@ -610,6 +610,14 @@ def test_transform_hf(generators, paulix_ops, paulix_sector, num_electrons, num_
             ),
             1,
         ),
+        (
+            ["H", "H", "H"],
+            np.array(
+                [[0.0, 0.0, 0.0], [0.0, 0.0, 1.0], [0.0, 0.0, 2.0]],
+                requires_grad=True,
+            ),
+            0,
```

Comment:
We cannot handle handle open shell systems with our DHF yet. I have added an error message for that in an upcoming PR. You might change the test for H4 or use a linear H3+ if the latter still fails because of the wiremap issue.

Reply:
I have updated this to H4.